src/Ledger/Conway/Specification/Gov/Actions.lagda.md

```diff
@@ -91,121 +163,54 @@ GovActionData TreasuryWithdrawal  = Withdrawals
-
-\subsubsection{Table: Types of governance actions}
-\label{fig:types-of-governance-actions}
-
-\begin{tabular}{ll}
-\textbf{Action}  & \textbf{Description}\\
-\hline
-\endhead{}
-\NoConfidence{}            & a motion to create a \emph{state of no-confidence} in the current constitutional committee \\
-\UpdateCommittee{}         & changes to the members of the constitutional committee and/or to its signature threshold and/or terms \\
-\NewConstitution{}         & a modification to the off-chain Constitution and the proposal policy script \\
-\TriggerHardFork{}\footnotemark  & triggers a non-backwards compatible upgrade of the network; requires a prior software upgrade  \\
-\ChangePParams{}           & a change to \emph{one or more} updatable protocol parameters, excluding changes to major protocol versions (``hard forks'')\\
-\TreasuryWithdrawal{}            & movements from the treasury\\
-\Info{}                    & an action that has no effect on-chain, other than an on-chain record
-\end{tabular}
-
-\footnotetext{There are many varying definitions of the term ``hard fork'' in the blockchain industry. Hard forks typically refer
-  to non-backwards compatible updates of a network. In Cardano, we attach a bit more meaning to the definition by calling any upgrade that
-  would lead to \emph{more blocks} being validated a ``hard fork'' and force nodes to comply with the new protocol version, effectively
-  rendering a node obsolete if it is unable to handle the upgrade.}
-
-% \subsection{Voting and Ratification}
-% \label{sec:voting-and-ratification}
-% Every governance action must be ratified by at least two of these three bodies using their on-chain \defn{votes}.
-% The type of action and the state of the governance system determines which bodies must ratify it.
-% Ratified actions are then \defn{enacted} on-chain, following a set of rules (see \cref{sec:enactment,fig:enactment-types}).
-
-\subsection{Hash Protection}
-\label{sec:hash-protection}
-
-For some governance actions, in addition to obtaining the necessary votes,
-enactment requires that the following condition is also satisfied: the state
-obtained by enacting the proposal is in fact the state that was intended when
-the proposal was submitted.  This
-is achieved by requiring actions to unambiguously link to the state
-they are modifying via a pointer to the previous modification. A
-proposal can only be enacted if it contains the \GovActionId{} of the
-previously enacted proposal modifying the same piece of
-state.  \NoConfidence{} and \UpdateCommittee{} modify the same state, while
-every other type of governance action has its own state that isn't
+```
+-->
+
+## Hash Protection {#sec:hash-protection}
+
+For some governance actions, in addition to obtaining the necessary
+votes, enactment requires that the following condition is also
+satisfied: the state obtained by enacting the proposal is in fact the
+state that was intended when the proposal was submitted. This is
```

Comment:
I'd separate the condition to its own block to make it more clear (paragraph is way too long right now):

```suggestion
satisfied: 

> the state obtained by enacting the proposal is in fact the
> state that was intended when the proposal was submitted. 

This is
```